fragments/filter/README.md

```diff
@@ -1025,13 +1040,50 @@ filter=CASEI(provider) == 'NASA'
 {
   "filter-lang": "cql2-json",
   "filter": {
-    "op": "eq", 
+    "op": "=", 
     "args": [
       { 
-        "upper": { "property": "provider" }
+        "function" : "casei", 
+        "args" : [ { "property": "provider" } ]
       },
-      "NASA"
+      { 
+        "function" : "casei", 
+        "args" : [ "Straße" ]
+      }
     ]
   }
 }
-``` -->
+```
+
+### Example 13: Using the ACCENTI Accent-insensitive Comparison Function
+
+The predefined function `ACCENTI` allows for accent-insensitive comparisons. This function is
+defined in the Accent and Case-insensitive Comparison conformance class. In the example below,
+`ACCENTI('tiburon')` and `ACCENTI('tiburón')` evaluate to be equal.
+
+#### Example 13: cql2-text (GET)
+
+```http
+filter=ACCENTI(provider) = ACCENTI('tiburón')
```

Comment:
What's the reasoning to not include the ACCENTI functionality into CASEI?
From a German language perspective this is *really* weird as "Straße" works in CASEI, but "Süße"  (I assume) does not and I have to use ACCENTI (while ä does not contain an accent for us). ß and ü are both special characters in German, but are on the same level. It's hard to distinguish which to use when, especiialy as ü is not called an accent (like ú or so).

Reply:
Ah, found it: https://github.com/opengeospatial/ogcapi-features/pull/641#issuecomment-987853664

Reply:
I think two points here:
1) My understanding is that the unicode rules for case include ss/ß even though it's not really the same as other case changes like a/A. 
2) Accent here is used in a very general sense, rather than a precise technical one. I think technically, only acute and grave are accents, and circumflex and umlauts (and every other way to "modify" ASCII characters) fall into the larger category of diacritics.